src/frontend/src/handler/create_source.rs

```diff
@@ -87,6 +92,29 @@ async fn extract_json_table_schema(
     }
 }
 
+pub fn debezium_cdc_source_schema() -> Vec<ColumnCatalog> {
+    let debezium_columns = vec![("payload", DataType::Jsonb)];
```

Comment:
There seems to be no need for a `Vec` here? BTW, is it possible to be also implemented on `ColumnCatalog` like the followings?